packages/components/src/Switch/Switch.css

```diff
@@ -65,25 +90,42 @@
   padding-right: var(--space-small);
 }
 
+:global(.jobber-retheme) .label {
+  opacity: 0;
+}
+
 .pip {
   flex: 0 0 auto;
   width: var(--switch--pipSize);
   height: var(--switch--pipSize);
   border: var(--border-base) solid var(--color-border);
   border-radius: var(--radius-circle);
   background-color: var(--color-surface);
+  transition: all var(--timing-quick) ease-out;
+}
+
+:global(.jobber-retheme) .pip {
+  border: none;
+  background-color: var(--color-interactive--subtle);
 }
 
 .isChecked .pip {
   border-color: var(--color-green);
 }
 
-.disabled {
-  border-color: var(--color-disabled--secondary);
-  background-color: var(--color-surface--background);
-  cursor: not-allowed;
+:global(.jobber-retheme) .isChecked .pip {
+  background-color: var(--color-surface);
 }
 
+/* Moving this up overwrites disabled styling */
+/* stylelint-disable-next-line no-descending-specificity */
 .disabled .pip {
   border-color: var(--color-disabled--secondary);
 }
+
+.disabled,
+:global(.jobber-retheme) .disabled {
```

Comment:
I believe disabled have a different look when it is unchecked and disabled. For example, SG1

![image](https://github.com/GetJobber/atlantis/assets/15986172/dafbd234-1a07-4251-8b7b-76aa23a87bc3)

This CSS does this

![image](https://github.com/GetJobber/atlantis/assets/15986172/dbaaa1e2-f373-4367-996e-9d6e079e7d20)